src/o365/base/PeriodBasedReport.ts

```diff
@@ -21,16 +21,18 @@ export default abstract class PeriodBasedReport extends GraphCommand {
 
   public getTelemetryProperties(args: CommandArgs): any {
     const telemetryProps: any = super.getTelemetryProperties(args);
-    telemetryProps.period = args.options.period;
```

Comment:
While it doesn't make sense to track the usage of period here, where it's required, it seems like we're not tracking it at all in the DateAndPeriodBasedReport, where it's optional and where we should keep track if people request reports based on dates or periods.